src/overlays/actors/ovl_En_Sw/z_en_sw.c

```diff
@@ -481,23 +491,23 @@ void func_80B0D14C(EnSw* this, PlayState* play, s32 cnt) {
     }
 }
 
-void func_80B0D364(EnSw* this, PlayState* play) {
-    if (((this->actor.params & 0xE000) >> 0xD) == 4) {
-        this->unk_38C = 0;
-        this->actionFunc = func_80B0D3AC;
+void EnSw_SetupGoldHidden(EnSw* this, PlayState* play) {
+    if (ENSW_GET_GOLDTYPE(this->actor.params) == SW_GOLDTYPE_HIDDEN_TREE) {
+        this->waitTimer = 0;
+        this->actionFunc = EnSw_GoldHiddenReveal;
```

Comment:
`EnSw_SetupGoldHidden` / `EnSw_GoldHiddenReveal`

So is it hidden or revealed?

Reply:
"GoldHidden" refers to the Skulltulas found in soil and trees, since they share very similar code with minor deviations vs the other types.